Pikipick/Pikipick/MultipeerConnectivity/MultipeerDataModel.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  MultipeerStringTransition.swift
+//  Pikipick
+//
+//  Created by 황정현 on 2022/07/25.
+//
+
+import Foundation
+
+enum SendData: String, CaseIterable {
+    case question, vote, emoji
+}
+
+func sendDataTypeIs(identifier: String) -> SendData {
+    switch identifier {
+    case "QQQ":
+        return SendData.question
+    case "VVV":
+        return SendData.vote
+    default:
+        return SendData.emoji
+    }
+}
+
+// PeerID + 질문 -> 질문으로 변환
+func questionIs (data: String) -> String {
+    guard let strIndex = data.firstIndex(of: "|") else { return "NIL"}
+    let identifierLength = data.distance(from: data.startIndex, to: strIndex)
+    let question = data.substring(from: identifierLength+1, to: data.count - 1)
```

Comment:
```suggestion
    let question = data.substring(from: identifierLength + 1, to: data.count - 1)
```
identifierLength를 한번 더 선언하니까 이해하기 쉬워졌어요!
네이밍이 처음에 안와닿았는데, 읽어보니까 왜 이렇게 쓰셨는지 알것같아요 :)
+ 연산자 사이에 공백을 넣어주세요!

Reply:
연산자 공백 반영했습니다!